src/ripple/nodestore/impl/codec.h

```diff
@@ -235,6 +235,80 @@ nodeobject_decompress (void const* in,
         write(os, is(512), 512);
         break;
     }
+    case 5: // compressed v2 inner node
```

Comment:
Its purely a coincidence that the numbers used in codec match the numbers in the other code. I'm okay with skipping 4 though.